tests/e2e_http.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import os
+import getpass
+import time
+import logging
+import multiprocessing
+import shutil
+from random import seed
+import infra.ccf
+import infra.proc
+import infra.jsonrpc
+import infra.notification
+import infra.net
+import e2e_args
+
+from loguru import logger as LOG
+
+
+def run(args):
+    hosts = ["localhost"]
```

Comment:
Is there a specific reason we run this test with one node?

Reply:
Not really, signatures are disabled right now, so the extra votes are not really an issue.